.github/workflows/test.yml

```diff
@@ -34,7 +34,18 @@ jobs:
           java-version: ${{matrix.java}}
 
       - name: Run tests
-        run: ./gradlew test
+        env:
+           KBASE_CI_TOKEN: ${{ secrets.KBASE_CI_TOKEN }}
+           KBASE_CI_TOKEN2: ${{ secrets.KBASE_CI_TOKEN2 }}
+        run: |
+           cp test.cfg.example test.cfg
+           sed -i "s#^auth_token1 =.*#auth_token1 = $KBASE_CI_TOKEN#" test.cfg
```

Comment:
You test stuff live? Do these tokens expire? You might want to make a calendar event to update once in a while.

Reply:
They're managed by devops and last for 100 years

Reply:
At some point I want to stand up the auth server in testmode rather than use tokens, but... later

Reply:
This is how the old tests ran so I didn't want to change *too* much